src/main/java/bio/terra/stairway/Control.java

```diff
@@ -0,0 +1,44 @@
+package bio.terra.stairway;
+
+import bio.terra.stairway.exception.DatabaseOperationException;
+import java.util.List;
+
+/**
+ * Control class can be retrieved from the Stairway object and provides methods helpful
+ * for debugging and recovery.
+ */
+public class Control {
+  private final FlightDao flightDao;
+
+  public Control(FlightDao flightDao) {
+    this.flightDao = flightDao;
+  }
+
+  /**
+   * Lookup a flight without the completed time constraint
```

Comment:
Probably worth stating what the completed time constraint is and why it's useful to avoid it.